test/runtests.jl

```diff
@@ -14,9 +14,10 @@ ENV["GKSwstype"] = "100"
 # environment settings
 islinux = Sys.islinux()
 istravis = "TRAVIS" ∈ keys(ENV)
+isappveyor = "APPVEYOR" ∈ keys(ENV)
 datadir = joinpath(@__DIR__,"data")
-visualtests = !istravis || (istravis && islinux)
-if !istravis
+visualtests = (!istravis || (istravis && islinux)) && !isappveyor
```

Comment:
This line would become `visualtests = !CI || (istravis && islinux)` meaning we do visual tests only locally or in Tavis+Linux where the plots are somewhat reproducible compared to local settings.